<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegForm</class>
 <widget class="QWidget" name="AuthRegForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuthRegForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #444444;</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>470</height>
    </rect>
   </property>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
background-color: #FF4C2B;
color: white;</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="authButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
background-color: #FF4C2B;
color: white;</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regChange">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>370</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: None;
border: None;
color: grey;</string>
    </property>
    <property name="text">
     <string>Already have an account? Log in</string>
    </property>
   </widget>
   <widget class="QLabel" name="regLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 30);
border-radius: 7px;
color: white;
padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 30);
border-radius: 7px;
color: white;
padding-left: 10px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>User Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 30);
border-radius: 7px;
color: white;
padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="authChange">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: None;
border: None;
color: grey;</string>
    </property>
    <property name="text">
     <string>Create new account</string>
    </property>
   </widget>
   <widget class="QLabel" name="authLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>authButton</zorder>
   <zorder>regButton</zorder>
   <zorder>regChange</zorder>
   <zorder>regLabel</zorder>
   <zorder>emailEdit</zorder>
   <zorder>loginEdit</zorder>
   <zorder>passEdit</zorder>
   <zorder>authChange</zorder>
   <zorder>authLabel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
